packages/gmp/binary_tag_removal.patch

```diff
@@ -0,0 +1,51 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
```

Comment:
Did you already submit this patch as a PR to the GMP repo? Better to only apply patches to the versions where it is necessary (but since there are no released version yet, I guess it could also just be removed once it is merged).